src/query/expression/src/lib.rs

```diff
@@ -48,6 +48,7 @@ mod expression;
 pub mod filter;
 mod function;
 mod kernels;
+pub mod legacy;
```

Comment:
what about moving to `converts::bincode`?

Reply:
ok